### ISSUE TITLE
Getting router routes fails on node

### ISSUE THREAD

**Issue by meztez** [rstudio/plumber#701]
```R
a <- pr()
pr_get(a, "/__swagger__/", to_route("../__docs__/"))
pr_get(a, "/__swagger__/index.html", to_route("../__docs__/index.html"))
a$routes
 Error in node[[children[1]]] <- addPath(node[[children[1]]], children[-1],  : 
  cannot add bindings to a locked environment 
>
```

### Describe the problem in detail
Discovered while using `$routes` on a router with mounted redirects working on #694.

**Comment by meztez** [comment]
I have fix, just doing more test to see if it is the best fix.

**Comment by schloerke** [comment]
Fixed in #702
